src/nft/components/bag/Bag.tsx

```diff
@@ -282,7 +314,7 @@ const Bag = () => {
   return (
     <Portal>
       {!(isProfilePage && profilePageState === ProfilePageStateType.LISTING) ? (
-        <Column zIndex={isMobile || isOpen ? 'modalOverTooltip' : '3'} className={styles.bagContainer}>
+        <BagContainer raiseZIndex={isMobile || isModalOpen}>
```

Comment:
Can we move this wrapper above the listing conditional since it's the top level for both cases?